src/BenchmarksApps/TodosApi/appsettings.json

```diff
@@ -5,7 +5,6 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*",
```

Comment:
I thought we wanted this in our appsettings file. https://github.com/dotnet/aspnetcore/issues/47862